Maddori.Apple/Maddori.Apple/Global/UIComponent/FeedbackTypeButtonView.swift

```diff
@@ -0,0 +1,136 @@
+    }
+    
+    // MARK: - property
+    
+    private var selectedType: FeedbackType = .continueType
+    private let continueShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowRadius = 2
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private let stopShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowRadius = 1
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = 10
+        return view
+    }()
+    private lazy var continueButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Continue", for: .normal)
+        button.setTitleColor(.white100, for: .normal)
+        button.titleLabel?.font = .main
+        button.backgroundColor = .blue200
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 10
+        let buttonAction = UIAction { [weak self] _ in
+            self?.touchUpToSelectType(.continueType)
+        }
+        button.addAction(buttonAction, for: .touchUpInside)
+        return button
+    }()
+    private lazy var stopButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Stop", for: .normal)
+        button.setTitleColor(.gray600, for: .normal)
+        button.titleLabel?.font = .main
+        button.backgroundColor = .white100
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 10
+        let buttonAction = UIAction { [weak self] _ in
+            self?.touchUpToSelectType(.stopType)
+        }
+        button.addAction(buttonAction, for: .touchUpInside)
+        return button
+    }()
+    
+    // MARK: - life cycle
+    
+    private func render() {
+        self.addSubview(continueShadowView)
+        continueShadowView.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+        }
+        continueShadowView.addSubview(continueButton)
+        continueButton.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+        }
+
+        self.addSubview(stopShadowView)
+        stopShadowView.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+            $0.leading.equalTo(continueShadowView.snp.trailing).offset(11)
+            $0.centerY.equalTo(continueShadowView)
+        }
+        stopShadowView.addSubview(stopButton)
+        stopButton.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+            $0.leading.equalTo(continueButton.snp.trailing).offset(11)
+            $0.centerY.equalTo(continueButton)
+        }
+    }
+    
+    // MARK: - function
```

Comment:
p4;

```suggestion
    // MARK: - func
```
로 변경 가능할까요오..?? ^______^
컨벤션 통일을 위함입니다... 하하

Reply:
[68e685d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/68e685de40d10b3a26a9ce71d4f906e12b8adc8c)
수정해서 반영해두었습니다 !